src/components/Trade/Trade.tsx

```diff
@@ -15,17 +14,15 @@ export type TradeProps = {
 
 export const Trade = ({ defaultBuyAssetId }: TradeProps) => {
   const { getDefaultAssets } = useDefaultAssets(defaultBuyAssetId)
-  const [sellAsset, setSellAsset] = useState<Asset | undefined>()
```

Comment:
these state variables were redundant - we are setting the values in the form once we obtain the default assets